test/es-module/test-esm-loader-side-effect.mjs

```diff
@@ -0,0 +1,19 @@
+// Flags: --experimental-loader ./test/fixtures/es-module-loaders/loader-side-effect.mjs --require ./test/fixtures/es-module-loaders/loader-side-effect-require-preload.js
```

Comment:
Can we also include a test that uses worker modules?

Reply:
Done.